packages/plot/src/utils.js

```diff
@@ -138,3 +138,29 @@ export const isDomainRangeEqual = (graphProps, nextGraphProps) => {
     isEqual(graphProps.range, nextGraphProps.range)
   );
 };
+
+export const findLongestWord = label => {
+  var longestWord = (label || '').split(' ').sort(function(a, b) {
```

Comment:
use let

Reply:
Also, you should take in consideration the fact that this is an html element, not always a String.
(eg: `<i>Long Text</i>,` `<div><strong>A</strong></div>`)

Reply:
replace function (a, b) {...} with (a, b) => {...}

Reply:
ok